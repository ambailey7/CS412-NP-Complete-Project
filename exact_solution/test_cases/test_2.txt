10 45
0 1 359.651
0 2 795.202
0 3 470.454
0 4 848.936
0 5 125.468
0 6 192.836
0 7 839.814
0 8 363.145
0 9 478.113
1 2 715.557
1 3 270.153
1 4 408.263
1 5 789.173
1 6 802.27
1 7 27.667
1 8 130.429
1 9 908.842
2 3 697.843
2 4 69.236
2 5 129.071
2 6 898.961
2 7 791.823
2 8 99.552
2 9 873.397
3 4 677.227
3 5 281.953
3 6 530.418
3 7 526.604
3 8 330.924
3 9 943.242
4 5 97.35
4 6 547.163
4 7 486.967
4 8 591.651
4 9 830.997
5 6 741.12
5 7 480.102
5 8 191.749
5 9 400.137
6 7 291.935
6 8 899.891
6 9 517.387
7 8 939.97
7 9 739.384
8 9 234.545