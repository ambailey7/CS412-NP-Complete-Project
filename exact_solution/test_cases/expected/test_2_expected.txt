2043.3399999999997
4 2 8 9 0 6 7 1 3 5 4